### ISSUE TITLE
Document FHIRCast Support

### ISSUE THREAD

**Issue by rahul1** [medplum/medplum#4854]

**Comment by maddyli** [comment]
Some items to add to this document: https://www.medplum.com/docs/fhircast

What is FHIRCast? 

What can it be used for? 
- Realtime interaction between apps 
- AI Scribes -> publish between the microphone/text recorder and the EHR in real time
- Clinical operations -> synchronize workflows and context across multiple apps 

Link to FHIRCast demo

How to use FHIRcast in MedplumClient 
(include link here to the .NET FHIRCast) 
- examples of fhircastSubscribe, fhircastConnect, fhircastPublish, fhircastGetContext, fhircastUnsubscribe

**Comment by ThatOneBro** [comment]
Yeah I also need to finish up this demo at some point and move it into the main repo:
https://github.com/medplum/medplum-fhircast-demo-new
